db/seeds.rb

```diff
@@ -6,22 +6,51 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-# appartments = [
-#     {
-#         street:,
-#         city:,
-#         state:, 
-#         manager:,
-#         email:,
-#         price:,
-#         bedrooms:,
-#         bathrooms:,
-#         pets:,
-#         user_id:
-#     }
-# ]
+apartments = [
+    {
+        street:"8694 New Salem St UNIT 202", 
+        city:"San Diego",
+        state:"CA", 
+        manager:"Vani Bobba",
+        email:"vbob@test.com",
+        price:"$515,000",
+        bedrooms:2,
+        bathrooms:2,
+        pets: "true",
+        user_id: 1
```

Comment:
You can remove this user_id since you are assigning the apartment to a user on line 51